From 6695555e50a374f897965300568253f242a0b13b Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Fri, 23 Sep 2022 15:25:48 +0200
Subject: tune2fs: fix an error message

  $ tune2fs -O ^has_journal -ff /dev/sdh2
  Recovering journal. tune2fs: Unknown code ____ 251 while recovering journal.

  Before: Please run e2fsck -fy -O.
  After: Please run e2fsck -fy /dev/sdh2.

Note this doesn't fix the "Unknown code" message, just the "Please run
e2fsck" one.

Signed-off-by: Lubomir Rintel <lkundrak@v3.sk>
Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---
 misc/tune2fs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/tune2fs.c b/misc/tune2fs.c
index f566ed81..f3ce443c 100644
--- a/misc/tune2fs.c
+++ b/misc/tune2fs.c
@@ -3105,7 +3105,7 @@ _("Warning: The journal is dirty. You may wish to replay the journal like:\n\n"
 		if (retval) {
 			com_err("tune2fs", retval,
 				"while recovering journal.\n");
-			printf(_("Please run e2fsck -fy %s.\n"), argv[1]);
+			printf(_("Please run e2fsck -fy %s.\n"), device_name);
 			if (!fs)
 				exit(1);
 			rc = 1;
-- 
cgit 

